packages/react/src/examples/forms/validation.scss

```diff
@@ -4,13 +4,17 @@
 
   &__main-title {
     font-size: var(--fontsize-heading-xl);
-    line-height: var(--lineheight-m);
+    letter-spacing: -1.2px;
```

Comment:
It might be better to use helper-classes here as an example

Reply:
- [ ] Use helper class